src/_guides/bump-sh-tutorials/api-platform.md

```diff
@@ -13,17 +17,45 @@ The following assumes your local machine is configured with PHP and API Platform
 1. [Create and name](https://bump.sh/docs/new?utm_source=bump&utm_medium=content_hub&utm_campaign=getting_started) your first API documentation. Then, retrieve the name and token of this documentation from the _CI deployment_ settings page.
 
 2. Install the Bump.sh CLI with [npm](https://docs.npmjs.com/cli/v9/configuring-npm/install?v=true) as below, or use [alternative options](/help/continuous-integration/cli), with
-  ```bash
+  ```shell
   npm install -g bump-cli
   ```
 
 3. Launch your local server with [Docker](https://api-platform.com/docs/distribution/#using-the-api-platform-distribution-recommended) or [Symfony](https://api-platform.com/docs/distribution/#using-symfony-cli)
 
 4. Deploy your doc to Bump.sh with
-  ```bash
+  ```shell
   bump deploy https://localhost/docs.json \
     --doc my-documentation-name \
     --token my-documentation-token
   ```
 
 That's it! Enjoy the comfort of Bump.sh to browse through your API doc, and [customize it to your needs](/help/getting-started/quick-start#customization-options).
+
+## Enriching definition files
+
+By default, API Platform generates what can be considered a plain technical description of your API. Although it is robust and complete, it's not enough for people discovering your API without any upfront knowledge about it, access to the code, or ability to test it extensively. Such definition files require contextual information, and any appropriate adjustements.
+
+API Platform lets you override, enrich and filter the generated OpenAPI definition files, [using PHP](https://api-platform.com/docs/core/openapi/). 
```

Comment:
I've been trying to get folks to standardize on "OpenAPI description documents" as per this glossary, got a lot of various vendors agreeing and I've been following that for articles and guides written for you. https://github.com/openapi-contrib/glossary

Reply:
Thanks for that feedback! I realize that I'm actually not entirely in sync with our own glossary https://docs.bump.sh/help/getting-started/concepts/#api-definition, where I should end up talking about an "OpenAPI document".

It's getting closer to what you mention. Not sure if you suggest we should go one step further, but it's a wider review than just this PR then 😄

Reply:
ha so long as you're consistent with your own glossary then do whatever in this blog post, can worry about aligning the rest later.